en/08_Changelogs/5.4.0.md

```diff
@@ -166,6 +166,10 @@ The changes include:
 - [`TemplateParser`](api:SilverStripe\View\TemplateParser) has been deprecated. It will be renamed to `SilverStripe\TemplateEngine\TemplateParser`.
 - [`ElementalAreaController::removeNamespacesFromFields()`](api:DNADesign\Elemental\Controllers\ElementalAreaController::removeNamespacesFromFields()) has been deprecated. It will be removed without equivalent functionality to replace it.
 - [`BaseElement::updateFromFormData()`](api:DNADesign\Elemental\Models\BaseElement::updateFromFormData()) has been deprecated. It will be removed without equivalent functionality to replace it.
+- [`FormField::extendValidationResult()`](api:SilverStripe\Forms\FormField::extendValidationResult()) has been deprecated. It will be replaced by the `updateValidate` extension hook.
+- [`DateField.rawValue`](api:SilverStripe\Forms\DateField->rawValue) has been deprecated. It will be removed without equivalent functionality to replace it.
+- [`DatetimeField.rawValue`](api:SilverStripe\Forms\DatetimeField->rawValue) has been deprecated. It will be removed without equivalent functionality to replace it.
+- [`TimeField.rawValue`](api:SilverStripe\Forms\TimeField->rawValue) has been deprecated. It will be removed without equivalent functionality to replace it.
```

Comment:
These need to be updated to reflect the changes in https://github.com/silverstripe/silverstripe-framework/pull/11464

Reply:
Updated